jobs/notifications/IssuePublishedNotifyUsers.php

```diff
@@ -99,7 +99,7 @@ protected function createMailable(
         Issue $issue,
         User $recipient,
         EmailTemplate $template,
-        Notification $notification
+        Notification|\PKP\notification\Notification $notification
```

Comment:
I think there's a missing `use` statement here, not another class named `Notification`.

Reply:
Kind of but not exactly , we need to have both the `APP\notification\Notification` and `PKP\notification\Notification` as APP Notification has few const and when type hinting in method, we need PKP Notification . updated it .

Reply:
That's better, thanks!